features/step_definitions/journey/registration_steps.rb

```diff
@@ -41,9 +55,14 @@
   if @tier == "upper"
     @journey.carrier_type_page.submit(choice: :carrier_broker_dealer)
     @carrier = "carrier_broker_dealer"
+
+    if @organisation_type == "limitedCompany"
+      @companies_house_number ||= "00000000"
```

Comment:
The `||=` is a very useful thing. It basically say "If the variable on the left is `nil` or `false`, then set its value to the value on the right". In this case, make sure that if the variable has been settled up in previous steps it does not get overwritten, but if it has not then we use a default value that will avoid the failure of the test. This is because we might not always be interested in setting it to a specific something

Reply:
Also, this `default value` here must be changed for something that is actually valid :D

Reply:
Sounds good and I can see that the default value has been updated!